guides/sdk-docs/external-apis/calling-external-apis.md

```diff
@@ -0,0 +1,130 @@
+---
+title: Calling external APIs from an Iris App
+category: new
+parentDocSlug: external-apis
+---
+
+In most cases to call an external API you can just use http client in your app's code and do the requests. But there are
+two situations in which you will need to use Iris App SDK Proxy service:
+
+1. When the external API does not implement CORS or does not allow Trackunit as an origin and the browser will not allow
+   the call
+2. To use secrets (like API key) without hardcoding them in your application
+
+## Manifest requirements
+
+To use the Iris App SDK Proxy you need to add `api.iris_app_sdk.proxy.fetch` scope (to make calls using Proxy) and
+optionally `account.iris.app.store-secret` scope (to store secrets) to your APPs manifest. Additionally, you need to
+specify the domains that your app is allowed to call in the cspHeader `"connect-src"`. So your manifest could look like
+this:
+
+```ts
+const irisAppManifest: IrisAppManifest = {
+    // ommiting other config
+    cspHeader: {
+        ...defaultCspConfig(),
+        "connect-src": [...defaultCspConfig()["connect-src"], "https://dbc-1234-567.cloud.databricks.com"],
+    },
+    scopes: [
+        {scope: "api.iris_app_sdk.proxy.fetch", optional: false},
```

Comment:
I just realised we use underscore and dot to separate iris_app `iris_app` and `iris.app`. I suggest we streamline:
```suggestion
        {scope: "api.iris.app.proxy.fetch", optional: false},
```